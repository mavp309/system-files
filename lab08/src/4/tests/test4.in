8.12 -3.45 2.0 7.6
