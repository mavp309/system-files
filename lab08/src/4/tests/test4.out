11.9200 -2.4500
